app/models/bundle_context.rb

```diff
@@ -39,6 +47,14 @@ def progress_log_file
     Tempfile.new.path(id) #FIXME: (#78)
   end
 
+  def stageable_discovery
+    {}
+  end
+
+  def accession_items
```

Comment:
effectively empty method worries me - perhaps addressed in subsequent commit?

Reply:
we've got tickets for other similar methods below... i think we should ticket this one in a similar manner if it's not already covered.  and then we can also excise the associated conditional logic that depends on these values.

Reply:
#262